blocks/subscriptions-block/features/checkout/default.jsx

```diff
@@ -22,12 +23,38 @@ const Checkout = ({ customFields }) => {
 	const [isOpen, setIsOpen] = useState(initialState);
 	const [isComplete, setIsComplete] = useState(initialState);
 	const [accountSummary, setAccountSummary] = useState();
-
+	const [billingAddress, setBillingAddress] = useState({});
+	const [captchaToken, setCaptchaToken] = useState();
+	const [captchaError, setCaptchaError] = useState();
+	const [resetRecaptcha, setResetRecaptcha] = useState(true);
+	const [error, setError] = useState();
+	// eslint-disable-next-line
```

Comment:
Why the next line was disabled for eslint errors?

Reply:
Since we are not using order info right now. It's throwing no unused vars. We can remove when we start to use `order` variable.